pkg/volume/csi/csi_plugin.go

```diff
@@ -205,13 +205,13 @@ func (p *csiPlugin) Init(host volume.VolumeHost) error {
 	p.host = host
 
 	if utilfeature.DefaultFeatureGate.Enabled(features.CSIDriverRegistry) {
-		csiClient := host.GetCSIClient()
+		csiClient := host.GetKubeClient()
 		if csiClient == nil {
-			klog.Warning("The client for CSI Custom Resources is not available, skipping informer initialization")
+			klog.Warning("The client for CSI Resources is not available, skipping informer initialization")
```

Comment:
Can we actually return an error instead of logging an error and continuing now that the object is in-tree? Will this be able to handle an HA upgrade case where apiserver is older than the kube-controller-manager?

Reply:
Will fix the first one. Not sure how to answer the 2nd question.

Reply:
I think the HA scenario is fine. The apiserver that controller manager talks to can only be equal or greater version

Reply:
Ok